Jenkinsfile

```diff
@@ -761,12 +761,10 @@ def setupNode() {
 
 def getUniqueID(testId, useSha) {
   // This is used also for Kubernetes namespace and should not > 62 chars.
-  def date = new Date().format("yyMMddHHmm").toString()
-  def prefix = isReleaseQualification() ? 'rc-test' : 'test'
+  def prefix = isReleaseQualification() ? 'rc-' : ''
   def uuid = randomUUID().toString()
-  def sha = useSha ? "${GIT_SHA.take(7)}-" : ''
   def identifier = testId != '' ? "${testId}-" : ''
-  return "${prefix}-${sha}${identifier}${uuid.take(4)}-${date}"
+  return "${prefix}${identifier}${uuid.take(6)}"
```

Comment:
My concern is: if test is aborted, service may not get cleaned properly.  we will have some service leaking.   If 2, or 3 such leaking,  we will run out of 100.

Can we not random the service name?   each test use its own service name,  just make sure they are not sharing name.   same e2e test may run in different stage: e2e and release-qualify, make sure they are not using the same service name too